build/travis-deploy.sh

```diff
@@ -10,37 +12,33 @@ if [ "${TRAVIS_EVENT_TYPE}" = "pull_request" ]; then
       exit 0
     fi
     echo "Building/pushing PR$TRAVIS_PULL_REQUEST from $PR_USER"
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=amd64
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=amd64
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=arm64
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=arm64
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=arm
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=arm
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=s390x
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=s390x
-    make push IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=ppc64le
-    make clean IMG_TAG="PR$TRAVIS_PULL_REQUEST" GOARCH=ppc64le
+    for GOARCH in ${GOARCHES[@]}; do
+      make push IMG_TAG="$GOARCH-PR$TRAVIS_PULL_REQUEST" GOARCH=$GOARCH
+      make clean IMG_TAG="$GOARCH-PR$TRAVIS_PULL_REQUEST" GOARCH=$GOARCH
```

Comment:
On this line and the line above I would add the `{}` brackets to the variable expansion to make it a little more obvious your intent
```suggestion
      make clean IMG_TAG="${GOARCH}-PR${TRAVIS_PULL_REQUEST}" GOARCH="${GOARCH}"
```